src/github.com/matrix-org/dendrite/cmd/dendrite-client-api-server/main.go

```diff
@@ -50,24 +51,28 @@ func main() {
 		log.Fatalf("Invalid config file: %s", err)
 	}
 
-	log.Info("config: ", cfg)
```

Comment:
This isn't strictly related to the PR, but it was bugging me since it logs the ed25519 private key of the server,